cpanfile

```diff
@@ -47,6 +47,7 @@ requires 'Mojo::UserAgent';
 requires 'Mojolicious', '>= 8.42';
 requires 'Mojolicious::Lite';
 requires 'Net::DBus';
+requires 'Net::Domain';
```

Comment:
With "standard Perl module" I actually mean *core* Perl module. So there's no need to specify the module here (but supposedly also no harm in doing so).